cargo-shuttle/src/args.rs

```diff
@@ -143,6 +143,9 @@ pub struct RunArgs {
     /// port to start service on
     #[clap(long, default_value = "8000")]
     pub port: u16,
+    /// use router ip address instead of localhost
+    #[clap(long)]
+    pub router_ip: bool,
```

Comment:
I'm not sold on the name, but can't think of anything better either :sweat_smile:

Reply:
The only other things I can really think of are `network` or `local_ip`, but yeah to be honest I'm not huge on either of them and `router_ip` is not really that descriptive

Reply:
Honestly tempted to use `network` or `lan` as the flag since `router_ip` or `use_router_ip` isn't technically accurate anymore, what do you think?

Reply:
Hmm, I was thinking more like 'host' but don't like it. How about `external`?

Reply:
Ok that sounds like a good idea, I'll add that in then